docs/decisions/0071-multi-agent-orchestration.md

```diff
@@ -0,0 +1,620 @@
+---
+# These are optional elements. Feel free to remove any of them.
+status: { proposed }
+contact: { Tao Chen }
+date: { 2025-04-30 }
+deciders: { Ben Thomas, Mark Wallace }
+consulted: { Chris Rickman, Evan Mattson, Jack Gerrits, Eric Zhu }
+informed: {}
+---
+
+# Multi-agent Orchestration
+
+## Context
+
+The industry is moving up the stack to build more complex systems using LLMs. From interacting with foundation models to building RAG systems, and now creating single AI agents to perform more complex tasks, the desire for a multi-agent system is growing.
+
+With the recent GA of the Semantic Kernel Agent Framework, which offers a stable agent abstraction and support for multiple agent services, we are now able to build on top of it to create multi-agent systems. This will allow our customers to unlock even more complex scenarios.
```

Comment:
Should we be explicit that this is the GA of the "single agent abstraction/APIs?"